### PR TITLE
fix welcome example

### PR DESCRIPTION
**Pull request by skydan**

### CODE REVIEW

**Comment by tanner0101** [comment]
What is the error that this is fixing?

**Comment by skydan** [comment]
image

or am I missing something?

**Comment by tanner0101** [comment]
Oh! Dang great catch. Can we update the note to say `hello/:someString` instead? That part is the typo.

File: Resources/Components/SafetySection/SafetySection.jade [hunk 1]
```diff
@@ -7,7 +7,7 @@ div.safety-section
     div.code-blocks
         h3 Vapor's pattern matching makes routing a breeze.
         code-block.
-            app.get("hello", <div class="code-hint" data-title="Strings are the most general type, and will match all requests.">String</div>) { request, <div class="code-hint" data-title="name is the matched String.">name</div> in 
+            app.get("welcome", <div class="code-hint" data-title="Strings are the most general type, and will match all requests.">String</div>) { request, <div class="code-hint" data-title="name is the matched String.">name</div> in 
                 return "Nice to meet you, \(name)."
             }
         span.comment Responds to all get request to <code>welcome/:someString</code>.
```
Comment (tanner0101) [inline Resources/Components/SafetySection/SafetySection.jade]: `hello/:someString` here

**Comment by skydan** [comment]
Sure thing! Should be fine now.

**Comment by tanner0101** [comment]
Sweet thanks :)

**Pull request merged**